crates/matrix-sdk-qrcode/src/error.rs

```diff
@@ -30,9 +30,6 @@ pub enum DecodingError {
     /// The QR code data is using an unsupported or invalid verification mode.
     #[error("the QR code contains an invalid verification mode: {0}")]
     Mode(u8),
-    /// The flow id is not a valid event ID.
-    #[error(transparent)]
-    Identifier(#[from] ruma_common::IdParseError),
```

Comment:
I wonder if I'm forgetting something here. Most likely not, and other implementations are treating this as an opaque string as well.

Reply:
The ID is no longer parsed, so I think this error is no longer necessary and will never happen.